addons/overpressure/functions/fnc_firedEHBB.sqf

```diff
@@ -54,8 +54,8 @@ if (_distance < _backblastRange) then {
     if (isClass (configFile >> "CfgPatches" >> "ACE_Medical") && {([_unit] call EFUNC(medical,hasMedicalEnabled))}) then {
         [_unit, _damage, "body", "backblast"] call EFUNC(medical,addDamageToUnit);
     } else {
-        TRACE_1("",isDamageAllowed _x);
-        if (!isDamageAllowed _x) exitWith {}; // Skip damage if not allowed
+        TRACE_1("",isDamageAllowed _unit);
+        if (!isDamageAllowed _unit) exitWith {}; // Skip damage if not allowed
         _unit setDamage (damage _unit + _damage);
```

Comment:
Any reason this is an `exitWith` when it could be a `then` for simplicity?

Reply:
I like `exitWith` :)

Reply:
Same. But I like classical control structures even more.

Reply:
classical one makes too many indents

Reply:
I think I might have just been trying to minimize the git diff, which isn't really a good reason